content/guides/references/configuration.md

```diff
@@ -31,17 +68,17 @@ default values.
 
 | Option                 | Default                           | Description                                                                                                                                                                                  |
 | ---------------------- | --------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| `baseUrl`              | `null`                            | URL used as prefix for [`cy.visit()`](/api/commands/visit) or [`cy.request()`](/api/commands/request) command's URL                                                                          |
-| `clientCertificates`   | `[]`                              | An optional array of [client certificates](/guides/references/client-certificates)                                                                                                           |
-| `env`                  | `{}`                              | Any values to be set as [environment variables](/guides/guides/environment-variables)                                                                                                        |
-| `includeShadowDom`     | `false`                           | Whether to traverse shadow DOM boundaries and include elements within the shadow DOM in the results of query commands (e.g. [`cy.get()`](/api/commands/get))                                 |
+| `baseUrl`              | `null`                            | URL used as prefix for [`cy.visit()`](/api/commands/visit) or [`cy.request()`](/api/commands/request) command's URL.                                                                         |
```

Comment:
This needs updated to be a e2e specific configuration option. Setting at the root level will now throw an error.

Reply:
@jaffrepaul it looks like this might have ben missed as part of your work on https://github.com/cypress-io/cypress-documentation/pull/4350. Can you take a look?

Reply:
@cowboy Without looking thru a proper diff, there could still be some necessary changes in https://github.com/cypress-io/cypress-documentation/pull/4280 which has been around for awhile. Can you approve so we can merge and discuss an audit and the current state of affairs? We've had to refactor and update a number of things along the way so an audit is certainly a good idea.